pom.xml

```diff
@@ -915,6 +915,46 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <version>0.11</version>
+        <executions>
+          <execution>
+            <phase>verify</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <reportFile>${project.build.directory}/${project.build.finalName}.rat</reportFile>
+          <excludeSubProjects>false</excludeSubProjects>
+          <excludes>
+            <exclude>**/target/**/*</exclude>
+            <!-- Exclude java file (already checked by checkstyle -->
+            <exclude>**/*.java</exclude>
+            <!-- Exclude json -->
+            <exclude>**/user.avsc</exclude>
+            <!-- Exclude txt test resources -->
+            <exclude>**/test/resources/**/*.txt</exclude>
+            <!-- Exclude package-list -->
+            <exclude>**/package-list</exclude>
+            <!-- Git files -->
+            <exclude>**/.git/**/*</exclude>
+            <!-- IDEA files -->
+            <exclude>**/.idea/**/*</exclude>
```

Comment:
Does this mean we need to blacklist all files that might be generated as part of development? Seems like it could be a burden. For example: yesterday I was running example pipelines and saving output files inside dev directory. The existence of these one-off files would prevent me from building?  Can we whitelist or use only files checked into Git?

If we do need to do this, is there any way to consolidate this with .gitignore rules?